apis/nucleus/src/hooks/useAppSelections.js

```diff
@@ -80,18 +81,23 @@ function createAppSelections({ app }) {
   };
   return appSelections;
 }
-export default function useAppSelections(app) {
+export default function useAppSelections(app, selectionStore) {
   if (!app.session) {
     // assume the app is mocked if session is undefined
     return [];
   }
-  const [appSelectionsStore] = useAppSelectionsStore();
+  let store = selectionStore;
+  if (!store) {
+    store = useContext(InstanceContext).selectionStore;
+  }
+
+  const [appSelectionsStore] = store.useAppSelectionsStore();
```

Comment:
you should not have conditional hooks
(could still work if `!store` and `store.useAppSelectionsStore` never change change)

Reply:
I can likely work around it yea, think it was the first hook I changed. Then I kept it as it simplified the tests.